server/prisma/schema.prisma

```diff
@@ -0,0 +1,33 @@
+generator client {
+  provider = "prisma-client-js"
+}
+
+datasource db {
+  provider = "sqlite"
+  url      = env("DATABASE_URL")
+}
+
+model ChunkData {
+  hash      Bytes  @id
+  version   Int
+  data      Bytes
+  world     String
+  chunkX    Int
+  chunkZ    Int
+  timestamp BigInt
```

Comment:
I believe we should store the timestamp as an integer, in seconds. BigInts/bigints are unreliable in NodeJS in my experience (see Discord discussion). What do you think?